src/racer/nameres.rs

```diff
@@ -1091,15 +1084,12 @@ pub fn search_prelude_file(pathseg: &core::PathSegment, search_type: SearchType,
     let mut out : Vec<Match> = Vec::new();
 
     // find the prelude file from the search path and scan it
-    let srcpaths = match std::env::var("RUST_SRC_PATH") {
-        Ok(paths) => paths,
-        Err(_) => return out.into_iter()
+    if config::RUST_SRC_PATHS.is_empty() {
```

Comment:
This change makes the comment above inaccurate; can you please update it?

Reply:
Will do.